### ISSUE TITLE
Operation Daybreak Flash zero-day exploit deployed by the ScarCruft APT Group

### ISSUE THREAD

**Issue by beast-fighter** [aptnotes/data#199]
Release date: (06/17/2016)

Vendor: Kaspersky

Link: https://securelist.com/operation-daybreak/75100/

PDF:

Group: ScarCruft

Other:

**Comment by kbandla** [comment]
```
{'Date': '06/17/2018',
 'Filename': u'Kaspersky_Flash-zero-day-ScarCruft-APT-Group(06-17-2018).pdf',
 'Pdf': u'https://app.box.com/s/dj0libdgcbb232zyhj3fq1ehkqyti1xm',
 'SHA-1': '257650cc95d4f77ec70553ce9bbf0a4e393ea4e3',
 'Source': 'Kaspersky',
 'Title': 'Operation Daybreak',
 'Year': '2018'}
```

**Comment by michaeldavie** [comment]
This paper was incorrectly marked as 2018; the correct year is 2016.

**Comment by kbandla** [comment]
Fixed in 12a0d865f43382889910a518117ea17acb609b6c. Thanks @michaeldavie !
